sympy/physics/mechanics/actuator.py

```diff
@@ -990,3 +991,196 @@ def __repr__(self):
         return (f"{self.__class__.__name__}("
                 f"{self.linear_stiffness}, {self.nonlinear_stiffness}, {self.pathway}, "
                 f"equilibrium_length={self.equilibrium_length})")
+
+class CoulombKineticFriction(ForceActuator):
+    r"""Coulomb kinetic friction with Stribeck and viscous effects.
+
+    Explanation
+    ===========
+
+    This represents a Coulomb kinetic friction with the Stribeck and viscous effect,
+    described by the function:
+
+    .. math::
+        F = (\mu_k f_n + (\mu_s - \mu_k) f_n e^{-(\frac{v}{v_s})^2}) \text{sign}(v) + \sigma  v
+
+    where :math:`\mu_k` is the coefficient of kinetic friction, :math:`\mu_s` is the
+    coefficient of static friction, :math:`f_n` is the normal force, :math:`v` is the
+    relative velocity, :math:`v_s` is the Stribeck friction coefficient, and
+    :math:`\sigma` is the viscous friction constant.
+
+    The default friction force is :math:`F = \mu_k f_n`.
+    When specified, the actuator includes:
+
+    - Stribeck effect: :math:`(\mu_s - \mu_k) f_n e^{-(\frac{v}{v_s})^2}`
+    - Viscous effect: :math:`\sigma v`
+
+    Notes
+    =====
+
+    The actuator makes the following assumptions:
+
+    - The actuator assumes slip.
+    - The normal force is assumed to be a non-negative scalar.
+    - The resultant friction force is opposite to the velocity direction.
+
+    This actuator has been tested for straightforward motions, like a block sliding
+    on a surface.
+
+    Examples
+    ========
+
+    The below example shows how to generate the loads produced by a Coulomb kinetic
+    friction actuator in a mass-spring system with friction.
+
+    >>> import sympy as sm
+    >>> from sympy.physics.mechanics import (dynamicsymbols, ReferenceFrame, Point,
+    ...     LinearPathway, CoulombKineticFriction, LinearSpring, KanesMethod, Particle)
+
+    >>> x, v = dynamicsymbols('x, v', real=True)
+    >>> m, g, k, mu_k, mu_s, v_s, sigma = sm.symbols('m, g, k, mu_k, mu_s, v_s, sigma')
+
+    >>> N = ReferenceFrame('N')
+    >>> O, P = Point('O'), Point('P')
+    >>> O.set_vel(N, 0)
+    >>> P.set_pos(O, x*N.x)
+    >>> P.set_vel(N, v*N.x)
+
+    >>> pathway = LinearPathway(O, P)
+    >>> friction = CoulombKineticFriction(mu_k, m*g, pathway, v_s=v_s, sigma=sigma, mu_s=mu_k)
+    >>> spring = LinearSpring(k, pathway)
+    >>> block = Particle('block', point=P, mass=m)
+
+    >>> kane = KanesMethod(N, (x,), (v,), kd_eqs=(x.diff() - v,))
+    >>> loads = friction.to_loads() + spring.to_loads()
```

Comment:
I would print `friction.to_loads()` out here as a doctest and to show the user what it looks like.